content/en/profiler/enabling/dotnet.md

```diff
@@ -61,14 +61,15 @@ The following profiling features are available in the following minimum versions
 | Wall time profiling       | 2.7.0+                             | All supported runtime versions.                                                          |
 | CPU profiling             | 2.15.0+                            | All supported runtime versions.                                                          |
 | Exceptions profiling      | 2.31.0+                            | All supported runtime versions.                                                          |
-| Allocations profiling     | Preview, 2.18.0+                   | .NET Framework beta (requires Datadog Agent 7.51+) and .NET 6+                           |
-| Lock Contention profiling | 2.49.0+                            | .NET Framework beta (requires Datadog Agent 7.51+) and .NET 5+                           |
-| Live heap profiling       | Preview, 2.22.0+                   | .NET 7+                                                                                  |
+| Allocations profiling     | beta, 2.18.0+                      | .NET Framework beta (requires Datadog Agent 7.51+) and .NET 6+                           |
```

Comment:
Somewhat unrelated, but just noticed in the context above `Supported .NET runtimes (64-bit applications)`. Is it work clarifying that this is only `x86_64`, _not_ `arm64`?

Reply:
This is a good point @andrewlock: I'm adding it to my TODO list for another dedicated PR. There might be other places where the update should be made